remote_backend/main.tf

```diff
@@ -0,0 +1,29 @@
+provider "aws" {
+    region = "eu-north-1"
+}
+
+# Create S3 bucket
+resource "aws_s3_bucket" "terraform_state" {
```

Comment:
This is not useful, it will not work because this bucket will not be created before terraform register the state. You need to create the bucket manually before the deployments

Reply:
commented it out. I will work on creating the bucket using aws cli before we run terraform apply.

Thank you for this.

Reply:
Amazing

Reply:
hmm. I was going to comment out the backend.tf file then run the script that creates the s3 bucket, then uncomment the backend script before running the terraform file for the other resources. Basically to ensure all the code is in a unified place. This is what I had in mind

Reply:
Either way works tho... but I used AWS CLI to create the S3 bucket. I also kept the code I used in the remote-state/backend.tf file.